### PR TITLE
Configfile, S&F and flooding

### PR DESCRIPTION
**Pull request by gjaskula99**
* Results are now rounded
* Configuration file implemented to overwrite default parameters
* MAC flooding implemented
* Interface speed can be changed to decide how many bytes should be proccessed at time
* Store&forward working with fixed frame size

Uniform and normal distros still throwing errors. Exponential works fine in certain range.

### CODE REVIEW

**Comment by gjaskula99** [comment]
Buffer queuing now fixed and overwrites frames correctly

**Comment by gjaskula99** [comment]
#4 now addressed

File: src/module-info.java [hunk 1]
```diff
@@ -1,2 +1,3 @@
 module L2SwitchSimulation {
+  requires java.desktop;
```
Comment (reksio6700) [inline src/module-info.java:2]: +1

File: src/l2/Simulator.java [hunk 2]
```diff
@@ -361,7 +396,93 @@ enum RNGTYPE {UNIFORM, EXP, NORMAL};
...
+	        rngType.setSelectedIndex( Integer.valueOf(prop.getProperty("TrafficRNG.type")) );
+	        System.out.println("TrafficRNG.param1 : " + prop.getProperty("TrafficRNG.param1"));
+	        generatorParam1 =  Double.valueOf(prop.getProperty("TrafficRNG.param1"));
+	        rngParam1.setText( prop.getProperty("TrafficRNG.param1") );
+	        System.out.println("TrafficRNG.param2 : " + prop.getProperty("TrafficRNG.param2"));
+	        generatorParam2 =  Double.valueOf(prop.getProperty("TrafficRNG.param2"));
+	        rngParam2.setText( prop.getProperty("TrafficRNG.param2") );
+	        
+		    System.out.println("Configuration loaded\n");
+		    assert BUFFERSIZE > 0;
```
Comment (reksio6700) [inline src/l2/Simulator.java:478]: Consider throwing errors instead of assertions

**Review by reksio6700 — APPROVED** [review]

**Pull request merged**